f2/src/components/formik/button/index.js

```diff
@@ -90,6 +94,14 @@ export const UploadButton = (props) => {
   )
 }
 
+export const FeedbackButton = (props) => {
+  return (
+    <Button type="submit" className="button feedback-button">
```

Comment:
The class name **feedback-button** is never defined. The result is a default Bootstrap button.

Reply:
I defined it in refactoring final feedback form, this code is probably from there.